docs/shared/checkpoint_learn_serialize.md

```diff
@@ -83,10 +83,10 @@ serialization of the class state.
 To serialize polymorphic class hierarchies, one must write serializers for each
 class in the hierarchy. Then, the user should either insert macros
 `checkpoint_virtual_serialize_root()` and
-`checkpoint_virtual_serialize_derived_from(T)` to inform *checkpoint* of the
+`checkpoint_virtual_serialize_derived_from(T)` to inform *magistrate* of the
```

Comment:
Note that `checkpoint_virtual_serialize` macros were never renamed to `magistrate_virtual_serialize`.

Reply:
Can we include this change in the PR?